src/Psalm/Config.php

```diff
@@ -164,13 +164,9 @@ class Config
 
     /**
      * These are special object classes that allow any and all properties to be get/set on them
-     * @var array<int, class-string>
+     * @var array<int, lowercase-string>
      */
-    protected $universal_object_crates = [
-        stdClass::class,
-        SimpleXMLElement::class,
-        SimpleXMLIterator::class,
-    ];
+    protected $universal_object_crates;
```

Comment:
I reworked this to avoid invoking `array_map('strlower'…)` on every call of a getter. I believe that BC check is actually false-positive as method's output didn't change.

Reply:
yeah, I don't think it's capable of detecting that :)

Reply:
BCC is valid (descendants are not guaranteed to call parent constructor), but we're in the alpha phase, and thus BC breaks are allowed.